sdk/search/tests.yml

```diff
@@ -5,9 +5,6 @@ jobs:
     parameters:
       ServiceDirectory: search
       Timeout: 120
-      # Live tests create their own resources for testing. Resources are created against a
-      # finite resource pool, so we limit parallelism to conserve availability in that pool.
-      MaxParallel: 3
```

Comment:
Did we want to remove this?

Reply:
We have so many tests which timeout for the second round. I want to test the behavior of removing this. If it works not like expected, we can add it back.

Reply:
There are 20 free quota under subscription. We only use 6 of them. Other languages sequentially executes on the same resources. Quickly finish our tests can minimize the conflicts with other languages.

Reply:
Sounds good, if the bucketing is working correctly it would be better for us to try to finish all our runs as fast as possible.